kernel/core/datamanagement/memory.cpp

```diff
@@ -841,7 +841,7 @@ void Memory::copyElementsInto(vector<mu::value_type>* vTarget, const VectorIndex
 bool Memory::isValidElement(size_t _nLine, size_t _nCol) const
 {
     if (_nCol < memArray.size() && _nCol >= 0 && memArray[_nCol])
-        return !mu::isnan(memArray[_nCol]->getValue(_nLine));
+        return memArray[_nCol]->isValid(_nLine); // THIS CHANGE MIGHT HAVE BROKEN OTHER THINGS

```

Comment:
Is this still relevant? Should this be removed or a TODO?

Reply:
Still relevant. Might yield problems in final tests and is therefore easier to find